Adding a note to update existing vulnerabilities indices mappings (#8104)
wazuh/wazuh-documentation: source/upgrade-guide/upgrading-central-components.rst

      # filebeat setup --pipelines
      # filebeat setup --index-management -E output.logstash.enabled=false

>    .. note:: When upgrading from v4.9.x, the ``wazuh-states-vulnerabilities-*`` mappings need to be updated manually with the command that follows:

Reviewer: ```suggestion
   .. note:: When upgrading from v4.8.x or v4.9.x, the ``wazuh-states-vulnerabilities-*`` mappings need to be updated manually with the command that follows:
```

> 
>    .. note:: When upgrading from v4.8.x or v4.9.x, the ``wazuh-states-vulnerabilities-*`` mappings need to be updated manually with the command that follows:

Reviewer 2: ```suggestion
#. If you are upgrading from versions v4.8.x or v4.9.x, manually update the ``wazuh-states-vulnerabilities-*`` mappings using the following command:
```
